### PR TITLE
Add first Orbit tutorial

### PR DESCRIPTION
**Pull request by PavelHolec**
An "hello world" tutorial. Wording is rough, any copy suggestions welcome 🙏 

How to run:
- open the package in XCode
- run Product > Build Documentation

image

### CODE REVIEW

**Review by sjavora — CHANGES_REQUESTED** [review]
Nice!

A note about screenshots - I _think_ we should have dark mode versions of them. Also, the "hello world" screen could potentially be made smaller - it's shown over the code sample (and I did not open it - it's done automatically).

Screenshot 2022-06-23 at 11 01 53

File: Sources/Orbit/Orbit.docc/Tutorials/Tutorials.tutorial [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@Tutorials(name: "Orbit") {
+    @Intro(title: "Meet Orbit") {
+        Meet Orbit, the open source design system for your next travel project.
+        
+        These tutorials will help you get familiar with our design system by building a simple login screen, while learning design foundations and components along the way.
```
[the remark this answers is not in the document]
Comment (PavelHolec) [reply Sources/Orbit/Orbit.docc/Tutorials/Tutorials.tutorial]: Removed for now. Any idea for some additional generic description?

**Comment by PavelHolec** [comment]
> A note about screenshots - I _think_ we should have dark mode versions of them. Also, the "hello world" screen could potentially be made smaller - it's shown over the code sample (and I did not open it - it's done automatically).

Resized the snapshots 👍 
Added dark variant, but for future tutorials, I am not sure it will be possible - as our components currently support only an experimantal, hacky, Dark Mode.

**Comment by sjavora** [comment]
True, and the only Xcode screenshots will probably be in this tutorial...

File: Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    @Section(title: "Create a new Orbit Enabled Project") {
+        @ContentAndMedia {
+            Create a new Xcode project that uses SwiftUI and has the Orbit package linked to it.
+        }
+        
+        @Steps {
+            @Step {
+                Create a new iOS app project in Xcode, using SwiftUI.
+                
+                @Image(source: 01-01-hello_orbit-s01, alt: "TBD")
```
Comment (sjavora) [inline Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial:19]:
```suggestion
                @Image(source: 01-01-hello_orbit-s01, alt: "A screenshot showing the App template selected in Xcode's project template picker")
```

File: Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            }
+            
+            @Step {
+                In Project settings, add a new swift package dependency.
+                
+                @Image(source: 01-01-hello_orbit-s02, alt: "TBD")
```
Comment (sjavora) [inline Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial:25]:
```suggestion
                @Image(source: 01-01-hello_orbit-s02, alt: "A screenshot of Xcode showing an open project with the Package Dependencies tab open, highlighting the plus button")
```

File: Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            }
+            
+            @Step {
+                Search for the Orbit package and select it. Specify a dependency rule. If you want to use the development version, choose the `main` branch. 
+                
+                @Image(source: 01-01-hello_orbit-s03, alt: "TBD")
```
Comment (sjavora) [inline Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial:31]:
```suggestion
                @Image(source: 01-01-hello_orbit-s03, alt: "A screenshot of Xcode showing the package search UI with orbit-swiftui shown in search results")
```

File: Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            }
+            
+            @Step {
+                In the Package Product selection screen, choose `Orbit`.
+                
+                @Image(source: 01-01-hello_orbit-s04, alt: "TBD")
```
Comment (sjavora) [inline Sources/Orbit/Orbit.docc/Tutorials/01-01-hello_orbit.tutorial:37]:
```suggestion
                @Image(source: 01-01-hello_orbit-s04, alt: "A screenshot of Xcode showing the package product selection pop-up with Orbit checked")
```

File: Sources/Orbit/Orbit.docc/Tutorials/Tutorials.tutorial (hunk 1 above)
[the remark this answers is not in the document]
Comment (sjavora) [reply Sources/Orbit/Orbit.docc/Tutorials/Tutorials.tutorial]: Not really...

**Review by sjavora — APPROVED** [review]

**Pull request merged**